conbench/entities/_comparator.py

```diff
@@ -1,18 +1,15 @@
 import decimal
 
+import sigfig
+
 from ..entities._entity import to_float
-from ..units import formatter_for_unit
 
 CHANGE = 5.0  # percent changed threshold
 Z_SCORE = 5.0  # z-score threshold
 
 
-def fmt(value):
-    return "{:.3f}".format(value) if value is not None else None
-
-
-def change_fmt(value):
-    return "{:.3%}".format(value)
+def _round(value) -> float:
```

Comment:
Could you add a type hint to the arg? Also I think the return value is optional here